apps/compositing/process.cpp

```diff
@@ -1,4 +1,5 @@
 #include <chrono>
+#include <cmath>
```

Comment:
Still won't work: https://stackoverflow.com/questions/6563810/m-pi-works-with-math-h-but-not-with-cmath-in-visual-studio

(╯°□°)╯︵ ┻━┻